README.md

```diff
@@ -2,7 +2,7 @@
 
 Simulate display output of a VGA core by running a testbench.
 
-Original VGA Simulator: https://www.ericeastwood.com/lab/vga-simulator/
+Original VGA Simulator: [https://www.ericeastwood.com/blog/vga-simulator-getting-started/](https://ericeastwood.com/blog/vga-simulator-getting-started/)
```

Comment:
You don't need to use the \[\]\(\) syntax here. Just replace it with the new link on its own: "https://ericeastwood.com..." without the brackets/parens.

Reply:
Thank you for the suggestion. I made appropriate changes. Kindly review them. This would be my first commit. Thank you for the support.